Makefile

```diff
@@ -33,6 +33,7 @@ ALL = \
 	$(ISO_ROOT)/salt/metalk8s/containerd/files/pause-$(PAUSE_IMAGE_TAG).tar \
 	\
 	$(ISO_ROOT)/salt/metalk8s/defaults.yaml \
+	$(ISO_ROOT)/salt/metalk8s/macro.sls \
```

Comment:
It's alphabetic order this should not be here :)

Reply:
yup